pyproject.toml

```diff
@@ -23,6 +23,9 @@ select = ["ALL"]
 pycodestyle.max-line-length = 120
 ignore = ["ANN101", "ANN102", "COM812", "D203", "D213", "S603"]
 
+[tool.ruff.lint.per-file-ignores]
+"tests/*" = ["S101"]
```

Comment:
There are a couple more coming in with a merge of the `issue_4` branch. If you wanted to get ahead of that I'm good with adding them in, now.